common/config/rush/pnpm-lock.yaml

```diff
@@ -120,7 +120,7 @@ dependencies:
   long: 4.0.0
   mocha: 5.2.0
   mocha-chrome: 1.1.0
-  mocha-junit-reporter: 1.23.0_mocha@5.2.0
+  mocha-junit-reporter: 1.23.1_mocha@5.2.0
```

Comment:
Does `pnpm-lock.yaml` need to be updated in this PR?  I don't see any changes to dependencies in `package.json`.

Reply:
reverted